# Common parameters
common:
  frame_id: "pandar"
  point_cloud_topic: "/driver/pandar/point_cloud"
  image_topic: "/camera/image_color"
  button_command_topic: "/lidar_camera_calibration/button_command"
  data_path: "/home/wxl/catkin_ws/src/lidar_camera_calibration/data/"
  lidar_chessboard_points_number: 100
  min_angle: 240.0
  max_angle: 300.0
  min_range: 2.0
  max_range: 100.0

# Data generate parameters
data_generate:
  starting_file_num: 0

# Lidar chessboard detector parameters
lidar_chessboard_detector:
  raw_cloud_topic: "/lidar_camera_calibration/raw_point_cloud"
  candidate_cloud_topic: "/lidar_camera_calibration/candidate_point_cloud"
  chessboard_cloud_topic: "/lidar_camera_calibration/chessboard_point_cloud"
  min_height: -1.7
  max_height: 1.0
  min_angle: 240.0
  max_angle: 300.0
  min_range: 2.0
  max_range: 5.0
  max_iterations: 100
  max_outlier_distance: 0.02
  cluster_tolerance: 0.2
  min_cluster_size: 10
  max_cluster_size: 5000

# Manual calibration parameters
manual_calibration:
  out_cloud_topic: "/lidar_camera_calibration/in_image_cloud"
  out_image_topic: "/lidar_camera_calibration/projected_image"
